packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconCritical from 'terra-icon/lib/icon/IconCritical';
+import IconHigh from 'terra-icon/lib/icon/IconHigh';
+import IconLow from 'terra-icon/lib/icon/IconLow';
+import IconAbnormal from 'terra-icon/lib/icon/IconAbnormal';
+import interpretationPropType from '../../proptypes/interpretationPropTypes';
+import valueQuantityPropType from '../../proptypes/valuePropTypes';
+import ResultError from '../other/_ResultError';
+import styles from './Observation.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  Event ID
+   */
+  eventId: PropTypes.string,
+  /**
+   *  Value and optional Unit of Measure for the Observation Result
+   */
+  result: valueQuantityPropType,
+  /**
+   * Interpretation of the Result, indicates Criticality
+   */
+  interpretation: interpretationPropType,
+  /**
+   * If Result has not been verified
+   */
+  isUnverified: PropTypes.bool,
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+};
+
+const defaultProps = {
+  eventId: undefined,
```

Comment:
We don't need to default props to undefined, so these can be removed

Reply:
abf3bfffa0813ec7f5066dac7f151a473b111cd0